README.md

```
@@ -371,6 +371,50 @@ tracker audit <run-id>
+})
+if result.Status == pipeline.OutcomeBudgetExceeded {
+    log.Printf("halt: %s, spent $%.4f", result.Cost.LimitsHit, result.Cost.TotalUSD)
+}
+for provider, pc := range result.Cost.ByProvider {
+    log.Printf("%s: %d tokens, $%.4f", provider, pc.Usage.InputTokens+pc.Usage.OutputTokens, pc.USD)
+}
+```
+
+**CLI users** pass flags directly to `tracker`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)